draft-ietf-quic-transport.md

```diff
@@ -1575,20 +1570,15 @@ Sending or receiving a STREAM frame causes the stream to become "open".  The
 stream identifier is selected as described in {{stream-identifiers}}.  The same
 STREAM frame can also cause a stream to immediately become "half-closed".
 
-An application can reserve an idle stream for later use.  The stream state for
-the reserved stream transitions to "reserved".
+Receiving a STREAM frame on a peer-initiated stream (that is, a packet sent by a
+server on an even-numbered stream or a client packet on an odd-numbered stream)
+also causes all idle streams with lower numbers to become "open".  This could
```

Comment:
This isn't accurate, since it's only odd/even streams that are marked open. How about '... also causes all lower-numbered idle streams in the same direction to become "open".' ?